docs/FAQ.md

```diff
@@ -537,13 +537,18 @@ commits associated with it.
 
 ### How do I integrate Jujutsu with Gerrit?
 
-At the moment you'll need a script, which adds the required fields for Gerrit
-like the `Change-Id` footer. Then `jj` can invoke it via an `$EDITOR` override
-in an aliased command. Here's an [example][gerrit-integration] from a
-contributor (look for the `jj signoff` alias).
+Add this to your configuration:
+```toml
+[templates]
+commit_trailers = '''
+if(
+  !self.trailers().filter(|t| t.key() == "Change-Id"),
```

Comment:
Now that #6425 is in, this can use the exact same form as https://jj-vcs.github.io/jj/latest/config/#commit-trailers, minus the Signed-off-by portion.